Fix for:

    src/afl-ld-lto.c:260:15: error: use of undeclared identifier 'PATH_MAX'
      char thecwd[PATH_MAX];

--- AFLplusplus-4.00c/src/afl-ld-lto.c
+++ AFLplusplus-4.00c/src/afl-ld-lto.c
@@ -45,10 +45,7 @@
 
 #include <dirent.h>
 
-#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || \
-    defined(__DragonFly__)
-  #include <limits.h>
-#endif
+#include <limits.h>
 
 #ifdef __APPLE__
   #include <sys/syslimits.h>
